Program Resources/Documentation Enhancement Grants README.md

```diff
@@ -0,0 +1,47 @@
+
+- **Comprehensive Developer Guides**: Develop detailed guides for developers covering various aspects of integrating Filecoin into their projects, including setup instructions, API usage, and troubleshooting tips.
+
+- **Interactive Learning Resources**: Develop interactive learning resources such as code playgrounds or interactive demos/videos that allow developers to experiment with Filecoin APIs and tools.
+
+## Program Qualifications
+Filecoin Documentation Enhancement Grants must meet the following criteria:
+
+- You have a clear plan to improve existing documentation or create new resources related to Filecoin.
+- You can provide a clear and straightforward description of the documentation enhancement project..
+- You can complete this work within 3 months from signing a grant agreement with Filecoin Foundation.
+- You agree to open-source the enhanced documentation, via a suitable license.
+- You agree to provide regular updates on the progress of your documentation enhancement project. This can be completed in Github.
+- If selected, grant applicants will need to provide additional information during the contracting and invoicing process.
+
+## How to Apply and Review Timeline
+To submit a proposal, [create a new issue](https://github.com/filecoin-project/devgrants/issues/new/choose) using the “Documentation Enhancement Grants" proposal template.
+
+We aim to notify Documentation Enhancement Grant applicants of the review decision by the end of the month following the application submission date.
+
+- For example, for applications submitted between March 1 and March 31, applicants will be notified by April 30.
+
+Our team may be in touch with questions in the proposal comment section in Github. **Please note that no work should begin without a signed contract.** 
+
+## Award
+Grants will be paid out once the following steps are completed:  
+- A grant agreement is signed by both parties
+- The outlined project work is complete
+  - As part of their final submission, all grantees should demonstrate successful usage by members of their intended audience with the intended audience as part of the project completion report.
+      - For example, for a tutorial, it should be tested by three to five developers who are not part of the creation team.
+- The project work has been reviewed and approved by Filecoin Foundation
+- Once a completed project has been reviewed and approved, Filecoin Foundation will make a single payment of up to $5,000 USD or USDC within 30 days.
+
+## FAQs & Support
+For more information regarding our grant program, including other grant types, check out [Filecoin DevGrants](https://github.com/filecoin-project/devgrants). For specific questions, you can reach our team directly by:
+- Email at grants@fil.org
+- Slack at [#grants-help](https://filecoinproject.slack.com/archives/C02A48BJ1SQ)
+- Joining our [Office Hours](https://calendly.com/filecoin-grants/office-hours-ama?month=2024-05).
```

Comment:
Edit the office hours like to remove month specification. Minor update to https://calendly.com/filecoin-grants/office-hours-ama

Reply:
Updated